pkg/config/cli_config.go

```diff
@@ -30,13 +30,8 @@ type CLIConfig struct {
 	// Server is the JVS server address.
 	Server string `yaml:"server,omitempty"`
 
-	// Authentication is the authentication config.
```

Comment:
⚠️ This is a change. I don't know why this was an embedded struct.

Reply:
I'm not a fan of a flat config and I was projecting a more complicated authentication config. We're using the application default credential to generate id token. Cloud Run accepts that. But not with IAP (we can forget about it now) or other authentication middleware (what if it requires an access token instead of id token). That's why I had the sub-struct. I could be overthinking.

Reply:
The reason I went with the flag config is because it's trivial to accidentally NPE if we're not careful. 

```go
var cfg config.CLIConfig

// later

if cfg.Authentication.Insecure { // npe
}
```

Since neither `Validate` nor `SetDefault` build the deeply-nested struct, it feels risky.

Reply:
It was initialized in SetDefault. But I think it's probably better to keep it simple now.